### DISCUSSION TITLE
V3: Register components using the Livewire::component

### DISCUSSION THREAD

**Discussion by nam-co** [livewire/livewire#5772] [7. Bugs (archived)]
On V3  when manually registering components using the Livewire::component, on composer update  I get an error: Non-static method Livewire\LivewireManager::component() cannot be called statically

Seems this page is still not available on the new docs: https://laravel-livewire.com/docs/2.x/package-dev#registering-components

Appreciate any help

**Comment by nuernbergerA** [comment]
try
```php
app('livewire')->component();
```

**Comment by nam-co** [comment]
@nuernbergerA Thanks, it seems in V2 you could  \Livewire::component is not working anymore, you need to "use Livewire\Livewire;" and then just use: Livewire::component to register the components

**Reply by nuernbergerA** [reply]
its fixed now
https://github.com/livewire/livewire/pull/5855
